frontend/src/console/components/CoursesManage/CoursesManage.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { InstancesModel } from 'console/models';
+import {
+  ConsolePage,
+  ConsolePageCustomizationContainer
+} from 'console/components';
+import { CONTACT_US_LINK, ROUTES } from 'global/constants';
+import { WrappedMessage } from 'utils/intl';
+import { Col, Button, Row } from 'react-bootstrap';
+import { RootState } from 'global/state';
+import { connect } from 'react-redux';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface State {
+  title: string;
+  subtitle: string;
+  // extra state to manage the empty title and subtitle and rendering
+  renderBool: boolean;
+}
+
+interface ActionProps {
+  clearErrorMessage: Function;
+  updateThemeFieldValue: Function;
+  updateImages: Function;
+}
+
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps, ActionProps {}
+
+export class CoursesManageComponent extends React.PureComponent<Props, State> {
+  public render() {
+    let instanceLink;
+    if (this.props.activeInstance.data) {
+      instanceLink = this.props.activeInstance.data.studioUrl;
+    } else {
+      instanceLink = ROUTES.Error.UNKNOWN_ERROR;
```

Comment:
@spokerman12 Final review comment here:
Can you disable the button styles instead of pushing the user to an error page if the Studio link isn't available?

Reply:
Sure thing. I replaced the error link with '#' and set disabled to check for '#' to avoid style hacking. 
Didn't use the button's href because it turns it into an <a>